query: |
  CALL migrate.postgresql('dummy_table', {
      host: 'localhost',
      port: 5432,
      user: 'memgraph',
      password: 'memgraph_password',
      database: 'test_db'
  })
  YIELD row
  RETURN row.range_col AS range_col

exception: true
